nncf/quantization/algorithms/accuracy_control/ranker.py

```diff
@@ -98,8 +98,9 @@ def find_groups_of_quantizers_to_rank(self, quantized_model_graph: NNCFGraph) ->
             if x.metatype in self._algo_backend.get_quantizer_metatypes()
         ]
 
-        quantized_model_graph_without_shapeof = remove_shapeof_subgraphs(
-            deepcopy(quantized_model_graph), self._algo_backend.get_shapeof_metatypes()
+        quantized_model_graph_without_shapeof = deepcopy(quantized_model_graph)
```

Comment:
This is just the graph copy, not `quantized_model_graph_without_shapeof`.

Reply:
Please see the next line

Reply:
I see and the `deepcopy(quantized_model_graph)` is still deepcopy, not `quantized_model_graph_without_shapeof`.